bee-network/bee-gossip/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bee-gossip"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Needs to be a minor since your are bumping actual deps

Reply:
Same for the others

Reply:
yea